starter-code/basic-algorithms.js

```diff
@@ -24,4 +20,31 @@ if (hacker1Length > hacker2Length ){
     console.log("wow, you both got equally long names,", hacker1Length, "characters!!")
   }
 
+
+// Palindrome Checker
+
+var palindromo = prompt("Please enter a palindrome");
+
+var revPal = palindromo.split("").reverse().join("");
+
+/* 
+  Intenté usar la función .equals pero no supe que estaba haciendo mal entonces mejor puse un switch
```

Comment:
aqui todo esta bien pero no funciona si coloco "anita lava la tina"
este es codigo algo mas sencillo el cual . nos permite tener un palindromo 

`var palindromo = prompt("Please enter a palindrome");

var revPal = palindromo.split(" ").join("");
let newWord = revPal.split("").reverse().join("")

let oldWord = palindromo.split(" ").join("")
if(oldWord === newWord){
  console.log("its a palindrome");
}else{
  console.log("not a palindrome");
}

`